prybar_assets/nodejs/input-sync.js

```diff
@@ -1,34 +1,19 @@
-const { readSync, writeSync, openSync } = require("fs");
-const { isatty } = require("tty");
+const { readSync, writeSync, openSync } = require('fs');
```

Comment:
Yikes, this whole project has inconsistent quoting styles, so we get these painful prettier diffs. I opened #97 and #98 as one-click merge options for how to try to mitigate this if you want.